objstore/objstore.go

```diff
@@ -0,0 +1,112 @@
+package objstore
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"fx.prodigy9.co/config"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+type StorageClient struct {
+	accessKey string
+	secretKey string
+	url       string
+	bucket    string
+	client    *minio.Client
+}
+
+var (
+	StorageAccessKeyConfig = config.Str("STORAGE_ACCESS_KEY")
+	StorageSecretKeyConfig = config.Str("STORAGE_SECRET_KEY")
+	StorageURLConfig       = config.Str("STORAGE_URL")
```

Comment:
I think the usual word is `_ENDPOINT` krub. (Other SDKs use this term as well)

Reply:
fixed krub